modules/xmpp/SdpConsistency.js

```diff
@@ -141,36 +118,28 @@ export default class SdpConsistency {
                 console.log("Sdp-consistency couldn't parse new primary ssrc");
                 return sdpStr;
             }
-            let newPrimaryRtxSsrc =
-                getRtxSsrc(videoMLine, newPrimarySsrc);
             if (!this.cachedPrimarySsrc) {
                 this.cachedPrimarySsrc = newPrimarySsrc;
-                this.cachedPrimaryRtxSsrc = newPrimaryRtxSsrc;
-                console.log("Sdp-consistency caching primary ssrc " +
-                    this.cachedPrimarySsrc + " and rtx " +
-                    this.cachedPrimaryRtxSsrc);
+                console.log("Sdp-consistency caching primary ssrc " + 
```

Comment:
I see that this is old code but I was wondering why we are using console.log instead of logger.log? Do you have any idea?

If there isn't reason for this maybe it's good moment to change them to logger since you are changing them. WDYT?